socorro/external/postgresql/signature_first_date.py

```diff
@@ -0,0 +1,40 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from socorro.external import MissingArgumentError
+from socorro.external.postgresql.base import PostgreSQLBase
+from socorro.lib import datetimeutil, external_common
+
+
+class SignatureFirstDate(PostgreSQLBase):
+    filters = [
+        ('signatures', None, ['list', 'str']),
+    ]
+
+    def post(self, **kwargs):
+        params = external_common.parse_arguments(self.filters, kwargs)
+
+        if not params['signatures']:
+            raise MissingArgumentError('signatures')
+
+        sql_params = [tuple(params['signatures'])]
```

Comment:
I'm stumped. How does this work? If I call it with `SignatureFirstDate(config).post(signatures='onestring')` where do you make that into a tuple?